### PR TITLE
RIP purpose and guidelines

### PR DESCRIPTION
**Pull request by nconsigny**
Adds an RIP that describes purpose and guidelines for the RIP process itself

### CODE REVIEW

File: RIPS/rip-XXX.md [hunk 1]
```diff
@@ -0,0 +1,599 @@
+---
+rip: XXX
```
Comment (abcoathup) [inline RIPS/rip-XXX.md:2]:
```suggestion
rip: 7810
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.

File: RIPS/rip-7810.md [hunk 2]
```diff
@@ -0,0 +1,600 @@
...
+
+:warning: If you are shepherding an RIP, you can make the process of building community consensus easier by making sure that [the Ethereum Magicians forum](https://ethereum-magicians.org/) thread for your RIP includes or links to as much of the community discussion as possible and that various rollups core teams are well-represented in those.
+
+*In short, your role as the champion is to write the RIP using the style and format described below, shepherd the discussions in the appropriate forums, and build community consensus around the idea.*
+
+### RIP Process
+
+The following is the standardization process for all RIPs in all tracks:
+
+![RIP RIP_process](../assets/rip-XXX/RIP_process.png)
```
Comment (CarlBeek) [inline RIPS/rip-7810.md:70]:
```suggestion
![RIP RIP_process](../assets/rip-7810/RIP_process.png)
```

This should match the RIP number, also the image directory that is being referenced here should also be updated to match

File: RIPS/rip-7810.md [hunk 3]
```diff
@@ -0,0 +1,600 @@
...
+
+**Living** - A special status for RIPs that are designed to be continually updated and not reach a state of finality. This includes most notably the present document.
+
+## What belongs in a successful RIP?
+
+Championing an RIP can be challenging due to the involvement of multiple stakeholders resulting from the plurality of rollups. One way to overcome this is to involve multiple rollups as early as possible in the discussion. Ideally, an author can co-write RIPs with multiple teams and ensure that the corresponding rollups are engaged.
+
+Each RIP should have the following parts:
+
+- Preamble - RFC 822 style headers containing metadata about the RIP, including the RIP number, a short descriptive title (limited to a maximum of 44 characters), a description (limited to a maximum of 140 characters), and the author details. Irrespective of the category, the title and description should not include RIP number. See [below](./rip-1.md#rip-header-preamble) for details.
```
Comment (CarlBeek) [inline RIPS/rip-7810.md:96]:
```suggestion
- Preamble - [RFC-822](https://www.w3.org/Protocols/rfc822/) style headers containing metadata about the RIP, including the RIP number, a short descriptive title (limited to a maximum of 44 characters), a description (limited to a maximum of 140 characters), and the author details. Irrespective of the category, the title and description should not include RIP number. See [below](./rip-1.md#rip-header-preamble) for details.
```

**Review by CarlBeek — APPROVED** [review]
LGTM, just some minor nits

**Pull request merged**